src/main/java/org/junit/Assert.java

```diff
@@ -989,11 +991,14 @@ public static void assertThrows(Class<? extends Throwable> expectedThrowable, Th
      * be obtained by calling {@link AssertionError#getCause}.
      *
      * @param expectedThrowable the expected type of the exception
-     * @param runnable       a function that is expected to throw an exception when executed
+     * @param runnable a function that is expected to throw an exception when executed
+     * @param message the identifying message for the {@link AssertionError} (<code>null</code>
+     * okay)
      * @return the exception thrown by {@code runnable}
      * @since 4.13
      */
-    public static <T extends Throwable> T expectThrows(Class<T> expectedThrowable, ThrowingRunnable runnable) {
+    public static <T extends Throwable> T assertThrows(Class<T> expectedThrowable,
+            ThrowingRunnable runnable, String message) {
```

Comment:
In JUnit 4.x the message comes first in the assertion methods, so perhaps we should do that to be consistent.

Besides consistency, the lambda have often be multiple lines, so I personally find it easier to read is the lambda is the last argument